src/main/java/com/patreon/euphrates/StreamParser.java

```diff
@@ -46,6 +46,9 @@ public void parse(InputStream is) {
       StringBuilder currentValue = null;
       while (reader.hasNext()) {
         int type = reader.next();
+
+        App.statsd.incrementCounter("bytes_processed", reader.getLocationInfo().getEndingByteOffset() - reader.getLocationInfo().getStartingByteOffset(), String.format("table:%s", table.name));
```

Comment:
Prefix the metric name with something about stream or parsing to help disambiguate?